django_inbound_email/backends/mailgun.py

```diff
@@ -43,9 +43,18 @@ def parse(self, request):
             )
             html = request.POST.get('stripped-html')
             from_email = request.POST.get('sender')
-            to_email = request.POST.get('recipient').split(',')
-            cc = request.POST.get('cc', '').split(',')
-            bcc = request.POST.get('bcc', '').split(',')
+            to_email = request.POST.get('recipient')
```

Comment:
I changed it to `None` because I couldn't do `if email.cc:` (`if email.cc:` on `['']` return `True` which I didn't expect). I've fixed tests accordingly now.